src/batteries_compattest.ml

```diff
@@ -54,6 +54,7 @@ module Stdlib_verifications = struct
 ##V>=4.14## end = struct
 ##V>=4.14##   include BatSeq
 ##V>=4.14##   let equal = equal_stdlib
+##V>=5.4##    let filteri = Stdlib.Seq.filteri
```

Comment:
Is there a reason to have it here just for compatibility, but not in `BatSeq` itself?